README.md

```diff
@@ -32,6 +32,12 @@ name: checker
 image: ghcr.io/openfaas/config-checker:latest
 namespace: openfaas
 sa: openfaas-checker
+# optionally specify resource requests/limits
```

Comment:
@alexellis what do you think about having a full example with all the currently supported parameters under `examples/` folder? In the readme I would just leave a link to the full example instead of copying all the newly added things.